docs benchmark (#1195)
PyTorchLightning/lightning-flash: docs/source/reference/image_classification.rst

    :lines: 14-

To learn how to view the available backbones / heads for this task, see :ref:`backbones_heads`.

> Moreover the backbone benchmarks of TIMM models can be found here:

Reviewer: Should we instead use `PyTorch Image Models` here? I was confused about what's TIMM.

```suggestion
Moreover, the benchmarks of different backbones of PyTorch Image Models (TIMM) can be found here:
```

> Moreover, the benchmarks of different backbones of PyTorch Image Models (TIMM) can be found here:

Reviewer 2: ```suggestion
Benchmarks for backbones provided by PyTorch Image Models (TIMM) can be found here:
```
